addons/packages/fluxcd-source-controller/0.21.2/package.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: fluxcd.source.controller.community.tanzu.vmware.com.0.21.2
```

Comment:
Should we keep the package name as source-controller.fluxcd.tanzu.vmware.com ?

Reply:
should we avoid the community on the name?

I can see why swap source-controller and fluxcd, I was just following the naming giving inside TAP
[TAP-source-controller](https://gitlab.eng.vmware.com/tap/tap-packages/-/blob/master/packages/fluxcd-source-controller/0.16.3/package.yaml#L6)

to keep some kind of consistency, also I added the community word just to keep align with other packages
[kpack](https://github.com/vmware-tanzu/package-for-kpack/blob/main/metadata.yaml#L7)

Reply:
so `fluxcd-source-controller.community.tanzu.vmware.com.0.21.2` instead of `fluxcd.source.controller.community.tanzu.vmware.com.0.21.2`